board/common/rootfs/usr/share/udhcpc/default.script

```diff
@@ -121,6 +121,17 @@ case "$ACTION" in
 	clr_dhcp_routes
 	set_dhcp_routes
 
+	# set hostname if given
+	if [ -n "$hostname" ]; then
+	    log "setting new hostname: $hostname"
+	    hostname "$hostname"
+
+	    if grep -q "^127.0.1.1.*" /etc/hosts; then
+		log "Updating /etc/hosts with new hostname"
+		sed -i "s/^\(127.0.1.1\s\+\).*/\1$hostname/" /etc/hosts
```

Comment:
Any utility in using `aug` here? (Like we do in `confd`)

Reply:
Heh, forgot about aug. I'll have a look, that would be useful also for one of our customer cases as well.

Reply:
Easy peasy with augtool :sunglasses: